docs/runbooks/LowReadyVirtOperatorsCount.md

```diff
@@ -0,0 +1,27 @@
+# LowReadyVirtOperatorCount 
```

Comment:
@yuhaohaoyu  I can't find this alert in the Kubevirt codebase. Is that a deprecated alert?

Reply:
Good catch. Its 'LowReadyVirtOperatorsCount', there is an 's'.

Reply:
@enp0s3 
Hi Igor, 

I fixed the type, great catch. Can you please go review the pull 45, https://github.com/kubevirt/monitoring/pull/45

It contains these changes. I will close this pull later. Sorry for the confusion.

Reply:
@yuhaohaoyu Sure np!